libs/novui/src/json-schema-components/JsonSchemaForm.tsx

```diff
@@ -46,18 +47,48 @@ export type JsonSchemaFormProps<TFormData = any> = JsxStyleProps &
   CoreProps &
   Pick<FormProps<TFormData>, 'onChange' | 'onSubmit' | 'onBlur' | 'schema' | 'formData' | 'tagName'> & {
     variables?: string[];
+    errors?: any;
   };
 
+const getExtraErrors = (errors: RJSFValidationError[]) => {
+  if (!errors) {
+    return {};
+  }
+  // adding a default error for the case of errors in an array not updating correctly
+  return { __errors: ['enableShowExtraErrors'], ...toErrorSchema(errors || []) };
+};
+
+// Always return an empty array to prevent the default error messages from showing
+function transformErrors(_: RJSFValidationError[]) {
+  return [];
+}
 /**
  * Specialized form editor for data passed as JSON.
  */
 export function JsonSchemaForm<TFormData = any>(props: JsonSchemaFormProps<TFormData>) {
-  const [cssProps, { className, variables, ...formProps }] = splitCssProps(props);
+  const [cssProps, { className, variables, errors, ...formProps }] = splitCssProps(props);
 
   const isAutocompleteEnabled = Boolean(variables && variables.length > 0);
 
+  const formRef = useRef<Form>(null);
+
+  function customValidate(formData: TFormData, formErrors: FormValidation<TFormData>) {
+    const extraErrors = getExtraErrors(errors);
+
+    if (Object.keys(extraErrors)?.length > 0) {
+      formErrors = { ...formErrors, ...extraErrors };
+    }
+
+    return formErrors;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Is mutating the `formErrors` object (param) the recommended pattern / way of accomplishing this?‏

Reply:
Took this from an example in their documentation